lib/MusicBrainz/Server/Data/Search.pm

```diff
@@ -741,8 +741,8 @@ sub external_search
         if (!$adv)
         {
             # Solr has a bug where the dismax end point behaves differently
-            # from edismax when the query size is 1. This is a fix for that
-            # See https://issues.apache.org/jira/browse/SOLR-12409
+            # from edismax (advanced) when the query size is 1. This is a fix
```

Comment:
I assume this comment is still relevant (that the advanced endpoint still uses the edismax query parser), but correct me if I'm wrong.